UPGRADE-4.0.md

```diff
@@ -25,6 +25,8 @@ If you have implemented a custom admin, you must adapt the signature of the foll
  * `getDashboardActions`
  * `getActionButtons`
  * `isCurrentRoute`
+ 
+The method signature of `configureActionButtons` has changed. A new parameter `actionList` was added.
```

Comment:
we should show old vs new here to make it easy to know on which position it was added